src/WallToolPaths.h

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) 2020 Ultimaker B.V.
+// CuraEngine is released under the terms of the AGPLv3 or higher.
+
+#ifndef CURAENGINE_WALLTOOLPATHS_H
+#define CURAENGINE_WALLTOOLPATHS_H
+
+#include <memory>
+
+#include "BeadingStrategy/BeadingStrategyFactory.h"
+#include "settings/Settings.h"
+#include "utils/ExtrusionLine.h"
+#include "utils/polygon.h"
+
+namespace cura
+{
+class WallToolPaths
+{
+public:
+    /*!
+     * A class that creates the toolpaths given an outline, nominal bead width and maximum amount of walls
```

Comment:
Offline remark: Is this going to replace `WallsComputation` then?

Reply:
Good question.
Maybe we should discuss this together with @rburema offline.
I will leave it be for now